app/controllers/admin/worldwide_offices_controller.rb

```diff
@@ -23,7 +23,7 @@ def update
 
     if (contact_numbers_attributes = update_params.dig(:contact_attributes, :contact_numbers_attributes))
       contact_numbers_attributes.each do |_id, contact_number_attribute|
```

Comment:
Could also do this in a separate commit to simplify this code, but it's not important!

https://github.com/alphagov/whitehall/pull/9084/files#r1624701069